src/ec/packages/ec-component-card/card.story.js

```diff
@@ -25,9 +26,28 @@ const formatInfo = data => {
 storiesOf('Components/Card', module)
   .addDecorator(withCode)
   .addDecorator(withNotes)
+  .addDecorator(withKnobs)
   .add(
     'card',
     () => {
+      dataCard.card.image.src = text('Image', dataCard.card.image.src);
```

Comment:
@Joosthe , at this point my question is: do we really want to find a way to overcomplicate things? :D
I understand that this works, but why did you decide to go this way instead of keeping things as they were just removing the "merge" that was the reason of the previous approach overheads? I mean, i really don't want to stress you, but i find this completely unreadable, while i was simply expecting you to create the card object (as it was before, as a key (string) value (from the specs) in the story, with the knob when you needed it, just without merging any pre-existing object from the specs because totally unneeded and with problematic consequences.
please tell me if there is anything unclear about this, we've been revolving this "improvement" enough, i think :)